storagemarket/impl/clientstates/cliest_states_test.go

```diff
@@ -36,16 +36,16 @@ func TestEnsureFunds(t *testing.T) {
 			},
 		}
 	}
-	t.Run("EnsureFunds succeeds", func(t *testing.T) {
+	t.Run("EnsureClientFunds succeeds", func(t *testing.T) {
 		runEnsureFunds(t, node(nil), nil, nil, nil, func(deal storagemarket.ClientDeal) {
-			require.Equal(t, deal.State, storagemarket.StorageDealFundsEnsured)
+			require.Equal(t, storagemarket.StorageDealFundsEnsured, deal.State)
 		})
 	})
 
```

Comment:
test of case where EnsureClientFunds gets MCID from node, goes to StorageDealClientFunding?